### DISCUSSION TITLE
Master Caution / Master Warning

### DISCUSSION THREAD

**Discussion by Jingles20** [nguyenquyhy/Flight-Tracker-StreamDeck#300] [Q&A]
Wondering if anyone has a toggle event string for the Master Caution / Master Warning buttons in most of the higher performance aircraft?  I've been working on my TBM 930 profile and would love the be able to assign a couple buttons to make those easier to disable.

**Comment by ccughostdk** [comment]
MOBIFLIGHT_TBM930_MASTER_WARNING_PUSH
MOBIFLIGHT_TBM930_MASTER_CAUTION_PUSH

MOBIFLIGHT_A32NX_MASTER_WARNING_PRESSED
MOBIFLIGHT_A32NX_MASTER_CAUTION_PRESSED

MOBIFLIGHT_WT_CJ4_MASTER_WARNING_PUSH
MOBIFLIGHT_WT_CJ4_MASTER_CAUTION_PUSH


Dont they do the trick.

**Reply by Jingles20** [reply]
Thanks man!

**Comment by Cotam57** [comment]
Hi,
Which variables are you using to display the master/caution warning in the TBM with Flight Tracker?

**Reply by ccughostdk** [reply]
You're kidding me aren't you? Try the first two lines, the ones that include TBM.

**Reply by Cotam57** [reply]
I'm not. 
If I understand correctly, these lines are events. They allow to perform an action. 
I'm looking for variables I can use to display the state of the switch. I tried using them but it does not work...maybe because they are events and not variables. I would like to have the two switches lighting as they are in the TBM cockpit.

**Reply by ccughostdk** [reply]
Try if these are usefull, but I´m not sure this is what you want:
MASTER_CAUTION_ACTIVE
MASTER_WARNING_ACTIVE
MASTER_CAUTION_ACKNOWLEDGED
MASTER_WARNING_ACKNOWLEDGED
If you turn on the brakes, the light will light up in the StreamDeck and you can turn off the light in the cockpit. But on your StreamDeck, the lamp will light up until you release the brakes.

**Reply by ccughostdk** [reply]
I personally use MASTER_CAUTION_ACTIVE and MASTER_WARNING_ACTIVE as feedback value. Then you have a reminder on the Streamdeck that something needs your attention. Eg. Inert Sep after take off.

**Reply by Cotam57** [reply]
That's what I wanna do but it does not work for me. Using MASTER_CAUTION_ACTIVE as a fb value returns 1 all the time....Maybe an installation issue, I dunno...

**Reply by Cotam57** [reply]
I confirm it does not work with the TBM and these variables are not referred in the SDK. Maybe it has been removed.

**Comment by Jingles20** [comment]
Maybe download this guy's AAO Streamdeck TBM profile?  He appears to have
Master Caution and Master Warning Lights working on his main page; wouldn't
the variables be the same?

https://flightsim.to/file/6109/tbm-930-streamdeck-icons-and-profile-for-lorbys-aao
